tests/benchmarks/benchmarks/mentat/license_update.py

```diff
@@ -0,0 +1,45 @@
+import importlib
+
+title = "License Update"
+
+description = """
+This benchmark tests the ability of Mentat to add a license to the allowed license list.
+"""
+
+prompts = [
+    # This one fails too by not putting HPND second. But it was the only prompt
+    # I could make that consistently passes verify.
+    'Add "HPND" second in the list of allowed licenses.',
+    # This one generally fails with an off by one error. Putting HPND
+    # outside of the list.
+    'Add "HPND" to the list of allowed licenses.',
+    # About half the time GPT spells out Historical Permission Notice and Disclaimer
+    "Add HPND to the list of allowed licenses.",
+]
+
+repo = "https://github.com/AbanteAI/mentat"
+commit = "b0848711c36e0c2fe9619ebb2b77dc6d27396ff2"
+minimum_context = ["tests/license_check.py:11-22"]
+
+
+def verify():
+    try:
+        import tests.benchmarks.repos.mentat.tests.license_check as license_check
+
+        importlib.reload(license_check)
+        return set(license_check.accepted_licenses) == set(
+            [
```

Comment:
To stop this from being flaky, why don't we get the list of licenses first instead of hard coding them here?

Reply:
Oh, I didn't realize that. Nevermind then!